src/test/utils/cashflow.test.js

```diff
@@ -44,4 +44,23 @@ test('Current SSDI cashflow 2', () => {
 	expect(cashflow.toCashflow(testClientList[0], 'current', 'SSDI')).toBe(99);
 });
 
+/** 
+* Test toCashFlow method.
+* For now, there are just test descriptions. 
+* These cannot be implemented until the client object is ready. 
+*/
+
+// Create a test client, setting new values for every prop for current values. 
+
+// Use toCashFlow to fetch each current property on the test client, confirming that 
+// it's the value we expect. 
+
+// Use toCashFlow to fetch an invalid current property on the test client, confirming that 
```

Comment:
The comments do look like comments, which is great! Is there a reason for the tests of invalid values, though, since client values should always be valid?

Reply:
For starters, thank you for reviewing and replying with comments!  Discussion is good. 
Here's what I'm thinking.  Is it helpful to flag an invalid property in case someone misspells it when passing to this routine?   I'm trying to anticipate future coding errors.  But I'll defer to you, whether you think this is helpful.

Reply:
Client values must always be valid, but if they're not toCashFlow is not the place that that should be dealt with. We can talk about it more detail, though, if you're still concerned.